main.py

```diff
@@ -1,7 +1,34 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
+
+genre_list = ["Western", "Action", "Dramma"]
+actor_lost = [("George", "Klooney"), ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"), ("Will", "Smith"),
+              ("Jaden", "Smith"), ("Scarlett", "Johansson")]
 
 
 def main() -> QuerySet:
-    pass
+    for genre in genre_list:
+        Genre.objects.create(name=genre)
+
+    for (name, name_last) in actor_lost:
```

Comment:
You really does not need a tuple
```suggestion
    for name, name_last in actor_lost:
```